beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -631,14 +627,20 @@ func winningCrosslink(state *pb.BeaconState, shard uint64, epoch uint64) (*pb.Cr
 	// Filter out shard crosslinks with correct current or previous crosslink data.
 	for _, a := range shardAtts {
 		cFromState := state.CurrentCrosslinks[shard]
-		h, err := hashutil.HashProto(cFromState)
+
+		cFromStateRoot, err := ssz.HashTreeRoot(cFromState)
```

Comment:
cFromStateRoot isn't a very meaningful name, can we improve it and make it clear that the c refers to a crosslink? A better name would help